share/chpharos/chpharos.sh

```diff
@@ -192,21 +194,110 @@
 
 CHPHAROS_VERSION=0.1.1
 
+if [ -z "${CHPHAROS_CFG}" ]; then
+  CHPHAROS_CFG="$HOME/.chpharosrc"
+fi
+
+# Load config
+if [ -f "${CHPHAROS_CFG}" ]; then
+  # shellcheck disable=SC1091,SC1090
+  . "${CHPHAROS_CFG}"
+fi
+
 if [ -z "${CHPHAROS_ROOT}" ]; then
   CHPHAROS_ROOT="$HOME/.pharos/chpharos"
 fi
 
-if [ -z "${CHPHAROS_VERSION_LIST_URL}" ]; then
-  CHPHAROS_VERSION_LIST_URL="https://raw.githubusercontent.com/kontena/chpharos/master/versions.config"
+if [ -z "${CHPHAROS_SVC_URL}" ]; then
+  CHPHAROS_SVC_URL="https://get.pharos.sh"
 fi
 
+if [ -z "${CHPHAROS_WEB_CLIENT}" ]; then
+  CHPHAROS_WEB_CLIENT="$(basename "$(command -v curl || command -v wget || echo "none")")"
+  [ "${CHPHAROS_WEB_CLIENT}" = "none" ] && _chpharos_error_echo "curl or wget required" && return 1
+fi
+
+CHPHAROS_UA="chpharos/${CHPHAROS_VERSION}+$(_chpharos_os)+${CHPHAROS_WEB_CLIENT}"
+
 PHAROS_VERSIONS=()
 
+_chpharos_write_token() {
+  [ -f "${CHPHAROS_CFG}" ]  && mv "${CHPHAROS_CFG}" "${CHPHAROS_CFG}.1" && grep -v CHPHAROS_TOKEN "${CHPHAROS_CFG}.1" > "${CHPHAROS_CFG}" && rm -f "${CHPHAROS_CFG}.1"
+  [ ! -z "$1" ] && echo "CHPHAROS_TOKEN=\"$1\"" >> "${CHPHAROS_CFG}"
+}
+
+_chpharos_login_curl() {
+  local username password
+  username="$1"
+  password="$2"
+  curl -sSL -XPOST "${CHPHAROS_SVC_URL}/auth" \
+    -d "username=${username}&password=${password}&grant_type=password" \
+    -H "Content-Type: application/x-www-form-urlencoded" \
+    -A "${CHPHAROS_UA}"
+}
+
+_chpharos_login_wget() {
+  local username password
+  username="$1"
+  password="$2"
+  wget "${CHPHAROS_SVC_URL}/auth" \
+    -O - \
+    --quiet \
+    --post-data="username=${username}&password=${password}&grant_type=password" \
+    -U "${CHPHAROS_UA}"
+}
+
+_chpharos_subcommand_login() {
+  echo "Log in using your Kontena Cloud credentials"
```

Comment:
Please change to `Login using your Kontena Account credentials`

Reply:
> When to Use Log in: If you’re using this term as a verb

Cloud changed to Account